ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts

```diff
@@ -1335,6 +1335,43 @@ const triggerSet: TriggerSet<Data> = {
         return { infoText: output.noDebuff!() };
       },
     },
+    {
+      id: 'TOP Sigma Superliminal/Blizzard',
+      // Omega-M casts Superliminal/Blizzard
+      // Track from Discharger (7B2E)
+      type: 'Ability',
+      netRegex: { id: '7B2E', source: 'Omega' },
```

Comment:
```suggestion
      netRegex: { id: '7B2E', source: 'Omega-M' },
```

Maybe this?

Reply:
I think you're correct here. The P2 is just Omega, but P5 here is Omega-M and the P2 and P5 npc are different sourceIDs.

P2:
```
|4000ED62|Omega|7B2E|Discharger|
```

P5 in same pull:
```
|4000EEB8|Omega-M|7B2E|Discharger|
|4000EEB8|Omega-M|7B2A|Superliminal Steel|4000EEB8|Omega-M|1.200|90.00|100.00|0.00|
```